sass/_jetpack.scss

```diff
@@ -8,37 +8,37 @@ Infinite Scroll
 body #infinite-handle { // increased specificity to override jetpack's css
 	padding: 50px 0;
 	text-align: center;
-	
-    span {
+
+	span {
 		@include button($color-blue);
-        
-        @media screen and (max-width: $width-phone) {
-            display: block;
-            margin: auto;
-            width: 90%;
-        }
+
+		@media screen and (max-width: $width-phone) {
+			display: block;
+			margin: auto;
+			width: 90%;
+		}
 	}
 }
 
 // full-screen overlay spinner
 body .infinite-loader { // increased specificity to override jetpack's css
-    height: 42px; // match height of button
-    margin: 50px auto;
-    
-    .spinner {
-        animation: rotate .7s infinite linear;
-        border: 8px solid #ddd;
-        border-radius: 100%;
-        border-top-color: $color-blue;
-        height: 42px;
-        margin: auto;
-        position: static !important; // overrides inline style
-        width: 42px !important; // overrides inline style
-        
-        > div {
-            display: none; // hide jetpack's default spinner parts
-        }
-    }
+	height: 42px; // match height of button
+	margin: 50px auto;
+
+	.spinner {
+		animation: rotate .7s infinite linear;
+		border: 8px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.